projects/1.16/assets/geolosys/geolosys/lang/zh_cn.json

```
@@ -1 +1,103 @@
-{}
\ No newline at end of file
+{
+  "block.geolosys.rhododendron": "杜鹃",
+  "block.geolosys.peat": "泥炭",
+  "block.geolosys.lignite_ore": "褐煤",
+  "block.geolosys.bituminous_coal_ore": "烟煤",
+  "block.geolosys.anthracite_coal_ore": "褐煤",
+  "block.geolosys.hematite_ore": "赤铁矿",
+  "block.geolosys.limonite_ore": "褐铁矿",
+  "block.geolosys.malachite_ore": "孔雀石",
+  "block.geolosys.azurite_ore": "蓝铜矿",
+  "block.geolosys.cassiterite_ore": "锡石",
+  "block.geolosys.teallite_ore": "硫锡铅矿",
+  "block.geolosys.galena_ore": "方铅矿",
+  "block.geolosys.bauxite_ore": "铝土矿",
+  "block.geolosys.autunite_ore": "钙铀云母",
+  "block.geolosys.platinum_ore": "铂",
+  "block.geolosys.sphalerite_ore": "闪锌矿",
+  "block.geolosys.coal_ore": "煤",
+  "block.geolosys.cinnabar_ore": "朱砂",
+  "block.geolosys.gold_ore": "金",
+  "block.geolosys.lapis_ore": "青金石",
+  "block.geolosys.quartz_ore": "杂石英",
+  "block.geolosys.kimberlite_ore": "金伯利岩",
+  "block.geolosys.beryl_ore": "绿柱石",
+  "block.geolosys.ancient_debris_ore": "古代遗迹",
+  "block.geolosys.nether_gold_ore": "贫金",
+  "block.geolosys.lignite_ore_sample": "褐煤样品",
+  "block.geolosys.bituminous_coal_ore_sample": "烟煤样品",
+  "block.geolosys.anthracite_coal_ore_sample": "褐煤样品",
+  "block.geolosys.hematite_ore_sample": "赤铁矿样品",
+  "block.geolosys.limonite_ore_sample": "褐铁矿样品",
+  "block.geolosys.malachite_ore_sample": "孔雀石样品",
+  "block.geolosys.azurite_ore_sample": "蓝铜矿样品",
+  "block.geolosys.cassiterite_ore_sample": "锡石样品",
+  "block.geolosys.teallite_ore_sample": "硫锡铅矿样品",
+  "block.geolosys.galena_ore_sample": "方铅矿样品",
+  "block.geolosys.bauxite_ore_sample": "铝土矿样品",
+  "block.geolosys.autunite_ore_sample": "钙铀云母样品",
+  "block.geolosys.platinum_ore_sample": "铂样品",
+  "block.geolosys.sphalerite_ore_sample": "闪锌矿样品",
+  "block.geolosys.coal_ore_sample": "煤样品",
+  "block.geolosys.cinnabar_ore_sample": "朱砂样品",
+  "block.geolosys.gold_ore_sample": "金样品",
+  "block.geolosys.lapis_ore_sample": "青金石样品",
+  "block.geolosys.quartz_ore_sample": "杂石英样品",
+  "block.geolosys.kimberlite_ore_sample": "金伯利岩样品",
+  "block.geolosys.beryl_ore_sample": "绿柱石样品",
+  "block.geolosys.ancient_debris_ore_sample": "古代遗迹样品",
+  "block.geolosys.nether_gold_ore_sample": "贫金样品",
+  "item.geolosys.iron_cluster": "铁簇",
+  "item.geolosys.gold_cluster": "金簇",
+  "item.geolosys.copper_cluster": "铜簇",
+  "item.geolosys.tin_cluster": "锡簇",
+  "item.geolosys.silver_cluster": "银簇",
+  "item.geolosys.lead_cluster": "铅簇",
+  "item.geolosys.aluminum_cluster": "铝簇",
+  "item.geolosys.uranium_cluster": "铀簇",
+  "item.geolosys.nickel_cluster": "镍簇",
+  "item.geolosys.platinum_cluster": "铂簇",
+  "item.geolosys.zinc_cluster": "锌簇",
+  "item.geolosys.osmium_cluster": "锇簇",
+  "item.geolosys.yellorium_cluster": "黄铀簇",
+  "item.geolosys.ancient_debris_cluster": "古代遗迹聚集",
```

|Feedback Priority|
|---|
|⛰ Mountain / **Blocking and requires immediate action**|

```suggestion
  "item.geolosys.ancient_debris_cluster": "远古残骸矿簇",
```